src/Pages/Home/Home.jsx

```diff
@@ -1,4 +1,5 @@
 import { Button, Speakers } from "../../Components";
+import Sponsors from "../../Components/Sponsors/Sponsors";
```

Comment:
Import sponsors from `../../Components`, the main advantage of importing and exporting from index.jsx is that we can coalesce the imports in one line